Sources/ComposableArchitecture/Store.swift

```diff
@@ -193,6 +194,32 @@ public final class Store<State, Action> {
     _ = self.send(action, originatingFrom: nil)
   }
 
+  /// Sends an action to the store with a given animation.
+  ///
+  /// See ``Store/send(_:)`` for more info.
+  ///
+  /// - Parameters:
+  ///   - action: An action.
+  ///   - animation: An animation.
+  @discardableResult
+  public func send(_ action: Action, animation: Animation?) -> ViewStoreTask {
```

Comment:
The main thing we'll want to discuss is whether or not it's appropriate to reuse `ViewStoreTask` here, or if we want to rename `ViewStoreTask` to `StoreTask` and deprecate the type alias to `ViewStoreTask`. That can probably come after the PR, though.

Reply:
Yeah I think especially given the long term prospects for `ViewStore` itself, `StoreTask` might be much better. Happy to go whichever way you folks prefer, and in this PR or a later one, so please do let me know which way you'd like to proceed whenever you're around!

Reply:
Hey guys, just checking in, any updates on this one? Would love to get it in

Reply:
Hey @HarshilShah! We almost certainly want this in, we just haven't had enough time to sync up on it yet with our schedules this month.

I think next steps, if you're up for it, would be to rename `ViewStoreTask` to `StoreTask` and deprecate `ViewStoreTask` as a type alias rename. That should bring the PR very close to being mergable.

Reply:
Gotcha, will do! Just gonna push up some commits here in a bit then

Reply:
Just pushed up the changes. I made the deprecation message mirror that from `WithViewStore` when set up with stateless stores. I haven't deprecated the `ViewStore.send` methods that return a ViewStoreTask though, should I do that as well?

Reply:
Thanks!

`ViewStore.send` is still fine to live, undeprecated, for a long time, and is still probably the majority use case over `Store.send`, we just want to return a consistent `StoreTask` type from both of them. The deprecation is just in case folks were referring to `ViewStoreTask` in their code bases, which is unlikely. I've pushed a small commit to address.

Reply:
Ah gotcha, makes sense!